guides/doc-Planning_Guide/topics/Provisioning_Concepts.adoc

```diff
@@ -62,41 +62,111 @@ For more information, see xref:figu-Satellite_Topology_with_Isolated_Capsule[].
 You can use HTTP booting to boot systems over a network using HTTP.
 
 [[http-booting-requirements]]
-==== HTTP Booting Requirements
+==== HTTP Booting Requirements with managed DHCP
 To provision machines through HTTP booting ensure that you meet the following requirements:
 
+.Client requirements
+
+* Ensure that all the network-based firewalls are configured to allow clients on the subnet to access the {SmartProxy}.
+For more information, see xref:figu-Satellite_Topology_with_Isolated_Capsule[].
+
+* Ensure that your client has access to the DHCP and DNS servers.
+
+* Ensure that your client has access to the HTTP UEFI Boot {SmartProxy}.
+
 .Network requirements
 
 * Optional: If the host and the DHCP server are separated by a router, configure the DHCP relay agent and point to the DHCP server.
 
+.{Project} requirements
+
+Although TFTP protocol is not used for HTTP UEFI Booting, {Project} use TFTP {SmartProxy} API to deploy bootloader configuration.
+
+* Ensure that both {ProjectServer} and {SmartProxy} have DNS configured and are able to resolve provisioned host names.
+
+* Ensure that the UDP ports 67 and 68 are accessible by the client so that the client can send and receive a DHCP request and offer.
+
+* Ensure that the TCP port 8000 is open for the client to download bootloader and Kickstart templates from the {SmartProxy}.
+
+* Ensure that the TCP port {smartproxy_port} is open for the client to download bootloader from the {SmartProxy} via HTTPS protocol.
+
+* Ensure that the host provisioning interface subnet has a DHCP {SmartProxy} set.
+
+* Ensure that the host provisioning interface subnet has a HTTP Boot {SmartProxy} set.
+
+* Ensure that the host provisioning interface subnet has a TFTP {SmartProxy} set.
+
+* Ensure that the host provisioning interface subnet has a Templates {SmartProxy} set.
```

Comment:
Can you rewrite this, like this: 

Ensure that the subnet that functions as the host's provisioning interface has the following requirements:

* a DHCP {SmartProxy}
* a HTTP Boot {SmartProxy}
*  a TFTP {SmartProxy}
* a Templates {SmartProxy}

Reply:
Sure, do you think this is the correct syntax:

```
* Ensure that the subnet that functions as the host's provisioning interface has the following requirements:
+
** a DHCP {SmartProxy}
** a HTTP Boot {SmartProxy}
** a TFTP {SmartProxy}
** a Templates {SmartProxy}
```

Reply:
No, double  bullets are the worst ;-) 
If you're OK with it, I'll rewrite this for you tomorrow and when you're back from PTO we can talk !